spec/features/user_deletes_attachment_from_a_coverage_spec.rb

```diff
@@ -0,0 +1,31 @@
+require "rails_helper"
+
+feature "User views attachment count and deletes an attachment from
+  a coverage", js: true do
+  scenario "successfully" do
+    coverage = create(:coverage)
+    attachment = create(:attachment, attachable: coverage)
+    attachment_count = coverage.attachments.size
+    subject_name = coverage.subject.title
+    user = user_with_assignments_access_to(coverage.course.department)
+
+    visit manage_assignments_course_path(coverage.course.id, as: user)
+
+    within(".class-card-assignment-control") do
+      expect(page).to have_content(attachment_count)
+    end
+
+    click_on t("manage_assignments.outcome_coverages.outcome_coverage.attachments-expandable-link")
+
+    within(".assignment-attachments") do
+      click_on t("manage_assignments.shared.attachments.delete_link")
+    end
+
+    expect(page).to have_content(subject_name)
+    expect(page).to have_no_content(attachment.file_file_name)
+
+    within(".class-card-heading") do
```

Comment:
Hmm...I'm not sure if there's any value in verifying that the expand link is still there. Can you explain why this might be useful?